server/src/rest/api/user.rs

```diff
@@ -0,0 +1,78 @@
+//! Rest API implementations of user-related operations
+/// openapi generated rest types
+pub use super::rest_types::*;
+use crate::grpc::client::GrpcClients;
+use axum::{extract::Extension, Json};
+use hyper::StatusCode;
+
+use svc_storage_client_grpc::prelude::*;
+
+/// Example REST API function
+#[utoipa::path(
+    post,
+    path = "/contact/signup",
+    tag = "svc-contact",
+    request_body = SignupRequest,
+    responses(
+        (status = 200, description = "Request successful.", body = String),
+        (status = 500, description = "Request unsuccessful."),
+    )
+)]
+#[cfg(not(tarpaulin_include))]
+// no_coverage: (Rnever) not unit testable, only integration tests
+pub async fn signup(
+    Extension(grpc_clients): Extension<GrpcClients>,
+    Json(payload): Json<SignupRequest>,
+) -> Result<Json<String>, StatusCode> {
+    rest_debug!("entry.");
+
+    let data: user::Data = payload.into();
+    let user_id = grpc_clients
+        .storage
+        .user
+        .insert(data)
+        .await
+        .map_err(|e| {
+            rest_error!("failed to insert user: {}.", e);
```

Comment:
add a `rest_debug!` entry to display the content of `SignupRequest`. This allows for easier debugging when needed

Reply:
https://github.com/aetheric-oss/svc-contact/pull/11/commits/59b2d52934224723bfa2ae6be1b62833f0e9a68a